runtime/jvmti/jvmtiThread.cpp

```diff
@@ -1325,7 +1397,12 @@ ownedMonitorIterator(J9VMThread *currentThread, J9StackWalkState *walkState, j9o
 	}
 
 	if ((NULL != obj)
-	&& (getObjectMonitorOwner(vm, obj, NULL) == targetThread)
+	&& (
+		(getObjectMonitorOwner(vm, obj, NULL) == targetThread)
+#if JAVA_SPEC_VERSION >= 24
+		|| isOwnedDetachedObjectMonitor(vm, targetThread, obj)
+#endif /* JAVA_SPEC_VERSION >= 24 */
+		)
 	&& !isObjectStackAllocated(walkState->walkThread, obj)
 	) {
```

Comment:
formatting can be improved by simple reordering of conditions
```suggestion
	&& !isObjectStackAllocated(walkState->walkThread, obj)
	&& ((getObjectMonitorOwner(vm, obj, NULL) == targetThread)
#if JAVA_SPEC_VERSION >= 24
		|| isOwnedDetachedObjectMonitor(vm, targetThread, obj)
#endif /* JAVA_SPEC_VERSION >= 24 */
	)) {
```

Reply:
this hasn't been addressed. why was this resolved?

Reply:
fixed the ordering previously, didn't realize still left the extra newline, now fixed.